src/main/java/lotto/LottoResult.java

```diff
@@ -0,0 +1,63 @@
+package lotto;
+
+import java.util.*;
+
+public class LottoResult {
+    private final Map<Integer, Integer> resultMap = new HashMap<>();
```

Comment:
컬렉션 이름과 동일한 이름을 사용하면 헷갈리지 않을까요??
```suggestion
    private final Map<Integer, Integer> results = new HashMap<>();
```

Reply:
맞습니다. 수정하도록 하겠습니다!